.github/workflows-source/schedule-daily.yml

```diff
@@ -132,6 +132,36 @@ jobs:
           ZH2_DLL01_CSV_SECRETS: "${{ secrets.ZH2_DLL01_CSV_SECRETS }}"
           ZH2_FILE_SHARE_KEY: "${{ secrets.ZH2_FILE_SHARE_KEY }}"
 
+  fi-tests-nightly:
```

Comment:
These seem like they're supposed to be part of a different PR?

Reply:
Yes, this was a messy rebase, sorry. I cleaned it up now.